docs/installation/homebrew.md

```diff
@@ -0,0 +1,22 @@
+# Homebrew
+
+- [⬅️️ Back](../installation.md)
+
+To install box using [Homebrew](https://brew.sh), you need to tap the box formula first.
+
+```bash
+brew tap flow-php/flow
+brew install flow
```

Comment:
I don't think this is actually installing flow-php, it seems it's installing that other `flow` tool available in the official distribution. 
To install our flow you would need to do `brew install flow-php/flow/flow`